README.md

```diff
@@ -43,8 +45,6 @@ psql -h localhost -p 5432
 
 Create .env file in server workspace based on the .env.example. See Deployment section for more information on the .env file.
 
-Also create .env file in client workspace based on the .env.example. 
```

Comment:
Actually this client .env file does exist and is required for the app to run! The example file is named confusingly (https://github.com/usdigitalresponse/usdr-gost/blob/main/packages/client/.env.development), I'll adjust that in a follow-on PR or feel free to rename it to env.example as part of this changeset.

Reply:
So FWIW I don't think the naming was confusing at all, but may require more familiarity with dotenv than some folks have (?). Dotenv will pick up any `.env` or `.env.$NODE_ENV` file, so it will pick up the existing `.env.development` file just fine without you needing to create a new `.env` (at least it did so quite reliably for me). Since you don't actually need to do anything here, I thought removing the instruction (which inaccurately references `.env.example` to boot) made the most sense.

Reply:
(If curious, this is a design choice inherited from the original Ruby dotenv gem, described here: https://github.com/bkeepers/dotenv#can-i-use-dotenv-in-production and you can see how NPM dotenv’s docs, which are newer, discourage this usage, described here: https://github.com/motdotla/dotenv#should-i-commit-my-env-file)

Reply:
@Mr0grog just tested building out the client without a .env file in my local build and the app doesn't load even when in development mode. It works when a .env file is present. 

I'm obviously not deeply familiar with dotenv and the mechanics here but wanted to flag that since I needed to add this file manually when doing my initial set up; perhaps there's a way to make this seamless so the next person to walk through this README can build successfully

Reply:
> Does it actually work for you locally? …just tested building out the client without a .env file in my local build and the app doesn't load even when in development mode.

Weird! It worked great for me. I never created a `.env` file.

Reply:
Maybe the best solution here is to rename the file to `.env.example` and retain the instruction. (As a side note, I generally don’t think it’s good to commit `.env` files with values that are intended to be used, so I think this would be better overall anyway!)

Reply:
🎉 That seems great. I wonder what is happening here for me locally 😬